src/web/components/Onwards/OnwardsTitle.tsx

```diff
@@ -5,39 +5,67 @@ import { palette } from '@guardian/src-foundations';
 import { headline } from '@guardian/src-foundations/typography';
 import { from } from '@guardian/src-foundations/mq';
 
+const linkStyles = css`
+    text-decoration: none;
+    color: ${palette.neutral[7]};
+
+    :hover {
+        text-decoration: underline;
+    }
+`;
+
+const TitleWrapper = ({
+    url,
+    children,
+}: {
+    url?: string;
+    children: JSXElements;
+}) =>
+    url ? (
+        <a className={linkStyles} href={url}>
+            {children}
+        </a>
+    ) : (
+        <>{children}</>
+    );
+
 export const OnwardsTitle = ({
     title,
     description,
+    url,
 }: {
     title: string;
     description?: string;
+    url?: string;
 }) => (
     <>
-        <h2
-            className={css`
-                ${headline.xsmall()};
-                color: ${palette.neutral[7]};
-                font-weight: 900;
-                padding-bottom: 14px;
-                padding-top: 6px;
-
-                ${from.leftCol} {
+        <TitleWrapper url={url}>
+            <h2
+                className={css`
                     ${headline.xsmall()};
+                    color: ${palette.neutral[7]};
                     font-weight: 900;
-                }
+                    padding-bottom: 14px;
+                    padding-top: 6px;
 
-                ${from.wide} {
-                    font-weight: 900;
-                }
-
-                margin-left: 10px;
-                ${from.leftCol} {
-                    margin-left: 0;
-                }
-            `}
-        >
-            {title}
-        </h2>
+                    ${from.leftCol} {
+                        ${headline.xsmall()};
+                        font-weight: 900;
+                    }
+
+                    ${from.wide} {
+                        font-weight: 900;
+                    }
+
+                    margin-left: 10px;
```

Comment:
I know this is existing code but is it worth lifting the `margin-left` up above the MQs and then you can put the `from.leftCol` `margin-left` in the first `from.leftCol` block?

Reply:
tbh, it's quite a lot of css to be in the component now, I'd lift it out into a const